openfga/v1/errors_ignore.proto

```diff
@@ -70,6 +70,7 @@ enum ErrorCode {
   invalid_authorization_model = 2056;
   unsupported_schema_version = 2057;
   cancelled = 2058;
+  invalid_start_time = 2059;
```

Comment:
Do we need more fine-grained error messages such as 
- Invalid format
- Time is in future
- No changes found in provided time

Things like that? But with much better naming than what I have mentioned here

Reply:
See generated validation here: https://github.com/openfga/api/pull/204/files#diff-a5b40c50f1319864a0e0fefe3720e613a942df6bd2a90abde44e83216dfab2fd